packages/core/src/domain/location-address.ts

```diff
@@ -3,3 +3,11 @@ import { Address } from "./address";
 // TODO add "coordinates" to AddressStrict as optional
 export type AddressStrict = Pick<Address, "addressLine2"> &
   Required<Omit<Address, "addressLine2" | "coordinates">>;
+
+export type AddressWithCoordinates = AddressStrict & { lat: string; lon: string };
```

Comment:
Now that this is being moved to the domain, it becomes something that might be used throughout the codebase - so it's important that's as compatible with other Address types as possible, to avoid back-and-forth converting from one type to another.

What do you think we update it to be something like this?

```ts
export type AddressWithCoordinates = AddressStrict & Required<Pick<Address, "coordinates">>;
```

I ended up coding this to see if that made sense, so if you want to I can send a commit (4 files).